### PR TITLE
Adding sync params to screenshot call.

### PR DESCRIPTION
**Pull request by prklm10**
- Adding sync params to screenshot call
- Timeout will be handled in CLI, will update the minimum version.

### CODE REVIEW

File: percy/providers/appAutomateProvider.js [hunk 1]
```diff
@@ -33,10 +33,12 @@ class AppAutomateProvider extends GenericProvider {
     scrollableXpath,
     topScrollviewOffset,
     bottomScrollviewOffset,
-    scrollableId
+    scrollableId,
+    sync
   } = {}) {
     let response = null;
     let error;
+    sync = sync || false;
```
Comment (chinmay-browserstack) [inline percy/providers/appAutomateProvider.js:41]: Remove default value

File: percy/providers/appAutomateProvider.js [hunk 2]
```diff
@@ -84,19 +87,21 @@ class AppAutomateProvider extends GenericProvider {
...
-  async percyScreenshotEnd(name, percyScreenshotUrl, statusMessage = null) {
+  async percyScreenshotEnd(name, percyScreenshotUrl, sync, statusMessage = null) {
     return await TimeIt.run('percyScreenshotEnd', async () => {
       try {
         await this.browserstackExecutor('percyScreenshot', {
           name,
           percyScreenshotUrl,
           status: percyScreenshotUrl ? 'success' : 'failure',
           statusMessage,
+          sync,
```
Comment (chinmay-browserstack) [inline percy/providers/appAutomateProvider.js:104]: Reason to add here?

File: index.js [hunk 3]
```diff
@@ -100,7 +102,8 @@ module.exports = async function percyScreenshot(driver, name, options = {}) {
         scrollableXpath,
         topScrollviewOffset,
         bottomScrollviewOffset,
-        scrollableId
+        scrollableId,
+        sync
```
Comment (chinmay-browserstack) [inline index.js:106]: Add return statement inside percyOnAutomate function as well

File: yarn.lock [hunk 4]
```diff
@@ -322,106 +322,106 @@
     "@nodelib/fs.scandir" "2.1.5"
     fastq "^1.6.0"
 
-"@percy/cli-app@1.27.0-beta.0":
```
Comment (chinmay-browserstack) [inline yarn.lock:325]: Remove this

File: percy/providers/appAutomateProvider.js (hunk 2 above)
Comment (prklm10) [reply percy/providers/appAutomateProvider.js:104]: this is an end call. There is a pr in selenium hub. This is for instrumentation.

File: percy/providers/appAutomateProvider.js (hunk 1 above)
Comment (prklm10) [reply percy/providers/appAutomateProvider.js:41]: for app automate we can have default value right? Removing it and adding null.

File: yarn.lock (hunk 4 above)
Comment (this-is-shivamsingh) [reply yarn.lock:325]: removed

File: index.js (hunk 3 above)
Comment (this-is-shivamsingh) [reply index.js:107]: done

**Review by Amit3200 — APPROVED** [review]

**Review by shantanuk-browserstack — APPROVED** [review]

**Review by ninadbstack — APPROVED** [review]
Blind lgtm

**Pull request merged**